extensions/objects/templates/ImageWithRegionsEditor.js

```diff
@@ -18,10 +18,10 @@
 
 // TODO(czx): Uniquify the labels of image regions
 oppia.directive('imageWithRegionsEditor', [
-  '$sce', '$compile', 'alertsService', '$document', 'explorationContextService',
-  'OBJECT_EDITOR_URL_PREFIX',
-  function($sce, $compile, alertsService, $document, explorationContextService,
-           OBJECT_EDITOR_URL_PREFIX) {
+  '$modal', '$sce', '$compile', 'alertsService', '$document',
```

Comment:
I know it was already like this, but could you switch $document and alertsService just so the pre-defined angular services are together? Thanks